src/github/tests/mod.rs

```diff
@@ -85,11 +87,11 @@ fn team_dont_add_member_if_invitation_is_pending() {
     let mut model = DataModel::default();
     let user = model.create_user("mark");
     let user2 = model.create_user("jan");
-    model.create_team(TeamData::new("admins").gh_team("admins-gh", &[user]));
+    model.create_team(TeamData::new("admins").gh_team(DEFAULT_ORG, "admins-gh", &[user]));
```

Comment:
you never call the gh_team method with an org which is different from `DEFAULT_ORG`. Do you plan to do this in the future? Otherwise, we can hardcode it, right?

Reply:
Well, everything was hardcoded before this PR :sweat_smile: With this PR I wanted to generalize it to make it simpler to write tests that work across orgs.